packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -5135,6 +5298,31 @@ Client authentication credentials for an [OAuth 2.0 Authorization Server (AS)](h
 }
 ```
 
+##### Apple Client Signing object
+
+The information is used to generate the secret JSON Web Token for the token requests to Apple IdP.
+
+| Property      | Description                                                                                                 | DataType | Nullable | Readonly | MinLength | MaxLength |
+| ------------- | ----------------------------------------------------------------------------------------------------------- | -------- | -------- | -------- | --------- | --------- |
+| privateKey    | The  PKCS #8 encoded private key that you created for the client and downloaded from Apple                  | String   | TRUE(Only Nullable in update operation and it will keep the existing value if it is null)    | FALSE    | 1         | 1024      |
+| kid           | The Key ID that you obtained from Apple when you created the private key for client                         | String   | FALSE    | FALSE    | 1         | 1024      |
```

Comment:
The Key ID that you obtained from Apple when you created the private key for the client